Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -6,9 +6,6 @@ import Foundation
 import UIKit
 import SnapKit
 import Shared
-import XCGLogger
```

Comment:
I decided to cleanup TabToolbar a lot. 
- Remove `SEL`
- Cleanup allocations
- Move the line drawing from Toolbar into TabToolbar